app/mailers/digest_mailer.rb

```diff
@@ -5,11 +5,10 @@
 ###
 
 class DigestMailer < ApplicationMailer
+  def digest(user, messages)
+    @messages = messages
+    return unless user.active?
```

Comment:
A log message might be good here. Is this a logical change that's distinct from the upgrade and linting?

Reply:
Oh, I think that's a mis-copy-paste, but actually, it is more correct.